x/bep3/keeper/keeper.go

```diff
@@ -40,11 +40,15 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 	return ctx.Logger().With("module", fmt.Sprintf("x/%s", types.ModuleName))
 }
 
+// ------------------------------------------
+//				Atomic Swaps
```

Comment:
These are centered in my IDE but spaced awkwardly on git. I like these section headers as they organize similar code into chunks. Does anyone have strong opinions for/against them?

Reply:
Related to https://github.com/Kava-Labs/kava/pull/370#discussion_r383744618

Reply:
I like headers as well. Though I don't see them much in other code, not sure if there's reasons against them I'm missing.